src/components/rectificationSummary/RectificationSummary.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Accordion, TextArea, TextInput } from 'hds-react';
+import ParkingFineSummary from '../parkingFineSummary/ParkingFineSummary';
+
+import './RectificationSummary.css';
+import { useTranslation } from 'react-i18next';
+
+const RectificationSummary = () => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <h2 className="rectification-summary-header">
+        Oikaisuvaatimuksen tiedot
+      </h2>
+      <div className="rectification-summary-container">
+        <div className="rectification-summary-details">
```

Comment:
Some spacing could be added between the text fields so they're easier to distinguish